railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -20,6 +20,13 @@ def edit
         require_application_and_environment!
 
         ensure_editor_available(command: "bin/rails credentials:edit") || (return)
+
+        if File.exist?("config/credentials.yml.enc") && !master_key_defined?
```

Comment:
I do not want to use filename directly (Because if we want to change the filename, there are more places to change). 
Since this path can be obtained from `EncryptedFile#content_path`, can use it as follows..

```diff 
-        ensure_master_key_has_been_added if Rails.application.credentials.key.nil?
+
+        if Rails.application.credentials.key.nil?
+          return say missing_credentials_message if Rails.application.credentials.content_path.exist?
+
+          ensure_master_key_has_been_added
+        end
```

Reply:
An error message to be showed when the same problem occurs with the `credentials:show` command already exists. 
https://github.com/rails/rails/blob/a9692941600c34afbc8d60d15cb5f179d30f67e5/railties/lib/rails/commands/credentials/credentials_command.rb#L72 

What do you think about reusing it?

Reply:
I totally agree with using it. Thanks for the suggestions including the previous.